process/security_baseline.md

```diff
@@ -0,0 +1,253 @@
+   * **Objective**: Ensure the security baseline is widely adopted.
+   * **Metrics**:
+     * Three OpenSSF software-based pilot projects adopt the baseline by 9/15/2024.
+     * All OpenSSF software-based projects adopt the baseline by the end of 2024.
+     * At least two other LF foundations adopt the baseline by the end of 2024.
+* **Reduction in Security Findings**
+  * **Objective**: Measure the effectiveness of the baseline in improving a project’s security posture.
+  * **Metrics**:
+    * Use Scorecard statistics to track the reduction of Critical, High, and Medium risk findings through the project lifecycle. Benchmark is established at the adoption time. 
+    * Monitor the number of exceptions at both project and foundation levels, expecting a decline as projects mature and become more secure.
+* **Improved OSS Maintainer and Consumer Experience**
+  * **Objective**: Enhance the experience for both OSS maintainers and consumers.
+  * **Approach**: Shift security left in the OSS supply chain to reduce downstream risks.
+  * **Shared Responsibility**: Encourage private sector consumers to support OSS security by funding or contributing developer hours.
+
+## Scope
+The security baseline SHALL apply to technical initiatives under the OpenSSF GitHub enterprise accounts. 
+
+## Constraints
+**Ecosystem support for tooling and configuration capabilities**: They vary depending on the programming languages and the roadmaps for new features or feature enhancements. 
+
+**Shortage of contributors for tooling development**:  Security in open source can only be achieved at scale with the right tool. Lack of easily adopted tools prevents us from achieving a higher level of security baseline. 
+
+**Maintainers' time strained by disproportionate open-source consumer demands**: The security baseline aims to enable more secure software development at speed. Nevertheless, the primary constraint remains the limited time available to maintainers, coupled with the demand for open-source software maintenance, including the enhancement of software security.
+
+## Basic Operating Principles 
+To navigate these constraints, the following operating principles are adopted:
+
+  * **Strong Bias Towards Automation and Automatability**
+    * **Objective**: Enhance security by default and position security as an enabler rather than an inhibitor.
+    * **Approach**: Prioritize automation and automatability in all processes.
+  * **Minimal, Achievable, and Practical Baseline Requirements**
+    * **Objective**: Design a security baseline that balances software reliability, performance, cost, and security.
+    * **Approach**:
+      * Ensure the baseline is minimal and achievable with current technology.
+      * Allow for incremental adoption throughout a software project’s lifecycle by shifting security left in the SDLC process.
+      * Reuse existing OpenSSF guides and technologies without imposing new requirements.
+  * **Continuous Improvement**
```

Comment:
Is this principle about continuous improvement of the security baseline? This was not immediately clear in the Objective and Approach.

Reply:
the three bullets under "Minimal, Achievable, and Practical Baseline Requirements" meant to ensure that the baseline is not adding additional burdens to the maintainers while adopting the baseline. The continuous improvement is more focused on  we reduce security burden on maintainers through continuous improvements,